### DISCUSSION TITLE
Deleted profile @ghost has achievements

### DISCUSSION THREAD

**Discussion by Technetium1** [community/community#31794] [Other Feature Feedback, Questions, & Ideas]
@ghost has achievements on the profile, this should not be the case.
![image](https://user-images.githubusercontent.com/9877150/188224467-1af0aba5-db80-4022-ba87-14feb0078bbe.png)

**Comment by radar-blip** [comment]
You should be happy for him or her. 😄

**Comment by jfullstackdev** [comment]
it's controlled by GitHub itself, so they can do anything that they think is good for it

**Comment by rromulos** [comment]
I don't see anything wrong considering that this profile has achieved achievements in the past, but If the account is under GitHub control for any illicit act then in my point this account should be disabled and hidden.

**Reply by jfullstackdev** [reply]
yes, Staff here can see this request,

but as far as I know, users that are deleted goes here, where it's just like keeping records of deleted users because it might be useful when needed, who knows

**Reply by rromulos** [reply]
@jdevstatic thanks for the explanation :)
